packages/@o3r/azure-tools/README.md

```diff
@@ -0,0 +1,42 @@
+<h1 align="center">Otter Azure tools</h1>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/AmadeusITGroup/otter/main/.attachments/otter.png" alt="Super cute Otter!" width="40%"/>
```

Comment:
this won't work with https://github.com/AmadeusITGroup/otter/pull/874

Reply:
this is merged on main not on release 10-next for now

Reply:
you are targeting `main` in the url

Reply:
good point 😄